### PR TITLE
bootloader_uefi: Update summary for a more descriptive text

### PR DESCRIPTION
**Pull request by emiura**
For usage with: https://confluence.suse.com/pages/viewpage.action?spaceKey=~martinsmac&title=Script+collecting+testcase+coverage

### CODE REVIEW

File: tests/installation/bootloader_uefi.pm [hunk 1]
```diff
@@ -7,8 +7,30 @@
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
-
```
Comment (dzedro) [inline tests/installation/bootloader_uefi.pm]: same as #7937

File: tests/installation/bootloader_uefi.pm [hunk 2]
```diff
@@ -7,8 +7,30 @@
...
+# "textmode=1" if VIDEOMODE="text")
+# - Call bootmenu_default_params (Edit grub parameters, add "Y2DEBUG=1",
+# graphics mode and resolution, serial output, console/log redirection)
+# - Call specific_bootmenu_params (Autoyast options, network options, debug
+# options, installer specific options, fips enablement, kexec parameters, addon
+# products, DUD loading, external ISO) unless it is is_caasp or is_jeos
+# - Call specific_caasp_params (STACK_ROLE)
+# - Save bootloader parameters in a screenshot
+# - if "USE_SUPPORT_SERVER" is defined, wait for mutex to unlock before sendind
+# "F10"
```
Comment (dzedro) [inline tests/installation/bootloader_uefi.pm]: Shouldn't be the additional description behind `# Summary & # Maintainer` ?

Comment (dzedro) [reply tests/installation/bootloader_uefi.pm]: Sorry I gues sit is betwen `# Summary and # Maintainer`

Comment (emiura) [reply tests/installation/bootloader_uefi.pm]: Well, from what I checked on other tests, the blank line is between copyright notice and summary only:

# notice and this notice are preserved.  This file is offered as-is,
# without any warranty.

# Summary: Start the installation process on s390x zVM using the z3270
#   terminal and an ssh connection
# Maintainer: xxxxxxxxxxxxxxxxx

**Pull request merged**